smpe/bld/service/current-hold.txt

```diff
@@ -0,0 +1,20 @@
+****************************************************************
+* Affected function: Zowe z/OSMF pre req                       *
+****************************************************************
+* Description: Install required z/OSMF PTF                     *
+****************************************************************
+* Timing: post-APPLY                                           *
+****************************************************************
+* Part: z/OSMF install                                         *
+****************************************************************
+
+Zowe API Mediation Layer Authentication service cannot function
+fully unless z/OSMF JWT support is available. Ensure PTFs
+associated with APAR PH12143: NEW FUNCTION - z/OSMF JWT support
+https://www.ibm.com/support/pages/apar/PH12143 are applied, 
+Applicable component levels
+R230 PSY UI66511   UP19/12/11 P F912  
```

Comment:
If there is no z/OS 2.2 PTF then Zowe must alter its minimum system requirements. I believe currently we document we work on z/OS 2.2 or higher. Talk this over with Joe Winchester and Bruce Armstrong. z/OS 2.2 is going out of support soon, but we made the support claim, so changing that might be tricky.